### DISCUSSION TITLE
Safari extension

### DISCUSSION THREAD

**Discussion by avk** [jordwest/news-feed-eradicator#382] [Ideas]
Works great in Chrome. Would love to see this in Safari. Anyone else?

**Comment by jordwest** [comment]
Maybe one day, if they decide to support [WebExtensions](https://stackoverflow.com/questions/47435082/will-safari-ever-support-webextensions-api)

**Comment by rvv-r** [comment]
Hi, Safari now support WebExtensions, but I think you need to distribute the extension through the Mac App Store.
https://developer.apple.com/documentation/safariservices/safari_web_extensions
